backend.py

```diff
@@ -105,11 +105,20 @@ def Next_time_func():
 parser.add_argument("--y", type=float)
 parser.add_argument("--z", type=float)
 parser.add_argument("--OM1_name", type=str)
-parser.add_argument("--OM1_value", type=float)
+parser.add_argument("--OM1_posX", type=float)
+parser.add_argument("--OM1_posY", type=float)
+parser.add_argument("--OM1_posZ", type=float)
+parser.add_argument("--OM1_dist", type=float)
 parser.add_argument("--OM2_name", type=str)
-parser.add_argument("--OM2_value", type=float)
+parser.add_argument("--OM2_posX", type=float)
+parser.add_argument("--OM2_posY", type=float)
+parser.add_argument("--OM2_posZ", type=float)
+parser.add_argument("--OM2_dist", type=float)
 parser.add_argument("--OM3_name", type=str)
-parser.add_argument("--OM3_value", type=float)
+parser.add_argument("--OM3_posX", type=float)
+parser.add_argument("--OM3_posY", type=float)
+parser.add_argument("--OM3_posZ", type=float)
+parser.add_argument("--OM3_dist", type=float)
```

Comment:
Maybe I need to remove this additions and just find it in the database using the name.